app/views/companies/show.html.erb

```diff
@@ -1 +1,13 @@
-<%= @company.organization %>
+<h1><%= @company.organization %></h1>
+<h4><%= @company.city %></h4>
+<h4><%= @company.state %></h4>
+
+<% if current_user.company? %>
```

Comment:
I agree.  This is something Alex and I discussed but I forgot to correct it.  Doing it now.